desktop/packages/mullvad-vpn/src/renderer/lib/components/animate/Animate.tsx

```diff
@@ -0,0 +1,14 @@
+import { AnimatePresentVertical, AnimatePresentVerticalProps } from './components';
+
+export type AnimateProps = {
+  type: 'present-vertical';
+} & AnimatePresentVerticalProps;
+
+export const Animate = ({ type, ...props }: AnimateProps) => {
+  switch (type) {
+    case 'present-vertical':
+      return <AnimatePresentVertical {...props} />;
+    default:
+      return null;
```

Comment:
In case you want to exhaust your `switch` and get a type-error when it is not:

```suggestion
      return type satisfies never;    
```

I know it goes against your `return null` but since it should never happen...